<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>826</width>
    <height>787</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>config配置</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox_RGB">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>200</height>
      </size>
     </property>
     <property name="title">
      <string>布料图片平均RGB值</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="0" column="0">
       <widget class="QPushButton" name="btnOPenini">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>初始预加载</string>
        </property>
       </widget>
      </item>
      <item row="0" column="6">
       <widget class="QLabel" name="labelReview">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>100</height>
         </size>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <layout class="QGridLayout" name="gridLayout_6">
        <item row="0" column="0">
         <widget class="QLabel" name="label_10">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string> R值 </string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_12">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string> B值 </string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_11">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string> G值 </string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="editR">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>46.6722</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="editG">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>101.0441</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="editB">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>178.0243</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="btnReview">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>预览</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_Fect">
     <property name="title">
      <string>缺陷种类及阈值</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="7">
       <widget class="QLineEdit" name="editFect3">
        <property name="text">
         <string>折痕</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="editThe13"/>
      </item>
      <item row="5" column="8">
       <widget class="QLineEdit" name="editThe15"/>
      </item>
      <item row="3" column="3">
       <widget class="QLineEdit" name="editFect8">
        <property name="text">
         <string>刮痕</string>
        </property>
       </widget>
      </item>
      <item row="2" column="7">
       <widget class="QLineEdit" name="editFect6">
        <property name="text">
         <string>异物</string>
        </property>
       </widget>
      </item>
      <item row="4" column="3">
       <widget class="QLineEdit" name="editFect11"/>
      </item>
      <item row="5" column="3">
       <widget class="QLineEdit" name="editFect14"/>
      </item>
      <item row="4" column="7">
       <widget class="QLineEdit" name="editFect12"/>
      </item>
      <item row="1" column="8">
       <widget class="QLineEdit" name="editThe3">
        <property name="text">
         <string>0.6</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QLineEdit" name="editFect2">
        <property name="text">
         <string>有胶水</string>
        </property>
       </widget>
      </item>
      <item row="0" column="8" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>判定阈值</string>
        </property>
        <property name="buddy">
         <cstring>editThe5</cstring>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLineEdit" name="editFect16"/>
      </item>
      <item row="6" column="7">
       <widget class="QLineEdit" name="editFect18"/>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="editThe4">
        <property name="text">
         <string>0.6</string>
        </property>
       </widget>
      </item>
      <item row="0" column="5" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>判定阈值</string>
        </property>
        <property name="buddy">
         <cstring>editThe2</cstring>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="editThe10"/>
      </item>
      <item row="0" column="7" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>缺陷种类</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>判定阈值</string>
        </property>
        <property name="buddy">
         <cstring>editThe1</cstring>
        </property>
       </widget>
      </item>
      <item row="3" column="5">
       <widget class="QLineEdit" name="editThe8">
        <property name="text">
         <string>0.6</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLineEdit" name="editFect1">
        <property name="text">
         <string>黑点杂质</string>
        </property>
       </widget>
      </item>
      <item row="4" column="8">
       <widget class="QLineEdit" name="editThe12"/>
      </item>
      <item row="1" column="5">
       <widget class="QLineEdit" name="editThe2">
        <property name="text">
         <string>0.6</string>
        </property>
       </widget>
      </item>
      <item row="0" column="6" rowspan="7">
       <widget class="Line" name="line_2">
        <property name="lineWidth">
         <number>4</number>
        </property>
        <property name="midLineWidth">
         <number>3</number>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item row="5" column="5">
       <widget class="QLineEdit" name="editThe14"/>
      </item>
      <item row="0" column="0" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>缺陷种类</string>
        </property>
       </widget>
      </item>
      <item row="3" column="8">
       <widget class="QLineEdit" name="editThe9">
        <property name="text">
         <string>0.6</string>
        </property>
       </widget>
      </item>
      <item row="6" column="8">
       <widget class="QLineEdit" name="editThe18"/>
      </item>
      <item row="6" column="5">
       <widget class="QLineEdit" name="editThe17"/>
      </item>
      <item row="2" column="5">
       <widget class="QLineEdit" name="editThe5">
        <property name="text">
         <string>0.6</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLineEdit" name="editFect7">
        <property name="text">
         <string>压痕</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>缺陷种类</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLineEdit" name="editFect10"/>
      </item>
      <item row="5" column="0">
       <widget class="QLineEdit" name="editFect13"/>
      </item>
      <item row="0" column="2" rowspan="7">
       <widget class="Line" name="line">
        <property name="lineWidth">
         <number>4</number>
        </property>
        <property name="midLineWidth">
         <number>3</number>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLineEdit" name="editFect4">
        <property name="text">
         <string>线痕</string>
        </property>
       </widget>
      </item>
      <item row="6" column="3">
       <widget class="QLineEdit" name="editFect17"/>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="editThe7">
        <property name="text">
         <string>0.6</string>
        </property>
       </widget>
      </item>
      <item row="3" column="7">
       <widget class="QLineEdit" name="editFect9">
        <property name="text">
         <string>无胶</string>
        </property>
       </widget>
      </item>
      <item row="5" column="7">
       <widget class="QLineEdit" name="editFect15"/>
      </item>
      <item row="1" column="1" alignment="Qt::AlignHCenter">
       <widget class="QLineEdit" name="editThe1">
        <property name="text">
         <string>0.6</string>
        </property>
       </widget>
      </item>
      <item row="4" column="5">
       <widget class="QLineEdit" name="editThe11"/>
      </item>
      <item row="2" column="8">
       <widget class="QLineEdit" name="editThe6">
        <property name="text">
         <string>0.6</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLineEdit" name="editThe16"/>
      </item>
      <item row="2" column="3">
       <widget class="QLineEdit" name="editFect5">
        <property name="text">
         <string>接头</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_Train">
     <property name="title">
      <string>训练相关设置</string>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>201</width>
        <height>81</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="1" column="1">
        <widget class="QLineEdit" name="editSave_inter">
         <property name="text">
          <string>10000</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="editMax_iteration">
         <property name="text">
          <string>30000</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>总训练次数</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_14">
         <property name="text">
          <string>保存间隔</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>30</y>
        <width>511</width>
        <height>81</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="1">
        <widget class="QLineEdit" name="editImgpath">
         <property name="text">
          <string>./testdata/train_images/</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="editXMLpath">
         <property name="text">
          <string>./testdata/train_images/</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="btnXML">
         <property name="text">
          <string>打开XML标记</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QPushButton" name="btnImg">
         <property name="text">
          <string>打开图片路径</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_Predict">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>150</height>
      </size>
     </property>
     <property name="title">
      <string>预测相关设置</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="2" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>保存文件名</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLineEdit" name="editSavename">
        <property name="text">
         <string>config_Blue.ini</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="pushButton_2">
        <property name="text">
         <string>结果保存路径</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>图片读取路径</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QRadioButton" name="radioButtonSave">
        <property name="text">
         <string>是否保存预测结果到本地</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2" colspan="2">
       <widget class="QLineEdit" name="lineEdit_2">
        <property name="text">
         <string>D:\codes\inferenceDST\</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2" colspan="2">
       <widget class="QLineEdit" name="lineEdit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>D:\codes\inferenceSRC\</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btnSave">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>保存</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>editThe1</tabstop>
  <tabstop>editThe2</tabstop>
  <tabstop>editThe5</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
